Tests/SentryTests/SentryKSCrashReportConverterTests.m

```diff
@@ -76,24 +76,23 @@ - (void)tearDown {
 //    XCTAssertEqualObjects(event.dist, @"201702072010");
 //}
 
-// TODO
```

Comment:
Can we maybe remove the other commented tests? We have git history to look them up, if we need them.

Reply:
I re-enabled them instead